src/validation/index.ts

```diff
@@ -45,4 +45,5 @@ export const validator = {
   overseasName,
   startingNew,
   trustInvolved,
+  updateBeneficialOwnerAndManagingOfficerType: updateBeneficialOwnerAndManagingOfficerType
```

Comment:
Can this be just:
updateBeneficialOwnerAndManagingOfficerType,

Reply:
Fixed